easybuild/easyconfigs/m/MDBM/MDBM-4.13.0-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'MDBM'
+version = '4.13.0'
+
+homepage = 'https://github.com/yahoo/mdbm'
+
+description = """MDBM is a super-fast memory-mapped key/value store"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/yahoo/%(namelower)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['99cec32e02639048f96abf4475eb3f97fc669541560cd030992bab155f0cb7f8']
+
+builddependencies = [
+    ('binutils', '2.39'),
+]
+
+dependencies = [
+    ('CppUnit', '1.15.1'),
+    ('libreadline', '8.2'),
+]
+
+skipsteps = ['configure']
+
+prebuildopts = 'sed -i -e "s/error/no-error/ ; s@/tmp/install@%(installdir)s@" Makefile.base && '
+prebuildopts += 'LDADD=-ldl'
+
+sanity_check_paths = {
+    'files': ['bin/mdbm_config', 'include/mdbm.h', 'lib64/libmdbm.so'],
+    'dirs': ['bin', 'include', 'lib64'],
+}
```

Comment:
@SimonPinches would you consider adding a `sanity_check_commands` step? Otherwise, it's fine.

Reply:
I've just added a `runtest` to the EB script.  It took over 10 mins on my system so if this is too slow there is also a `test-fast` if that's preferable.